invenio_app_ils/config.py

```diff
@@ -177,6 +177,10 @@ def _(x):
         "task": "invenio_app_ils.circulation.mail.tasks.send_overdue_loans_mail_reminder",
         "schedule": timedelta(days=1),
     },
+    "extend_active_loans_holiday": {
```

Comment:
I find this a bit dangerous maybe?
Probably better to discuss it IRL, but I would have instead probably triggered this new task (with a delay of 2/3 hours) after a change to the closures dates of a location, instead of running this every day.
Hopefully, the closures are updated very rarely...